shell/platform/embedder/fixtures/main.dart

```diff
@@ -45,8 +45,15 @@ void executableNameNotNull() {
   notifyStringValue(Platform.executable);
 }
 
+@pragma('vm:entry-point')
+void implicitViewNotNull() {
+  notifyBoolValue(PlatformDispatcher.instance.implicitView != null);
```

Comment:
What does this test do? To ensure that `PlatformDispatcher.instance.implicitView` isn't null? But this is only because we don't have an option to control it right? Should we add a TODO or at least comment to explain that this test might (should) be broken once we allow non-implicitView?

Reply:
This is just a fixture to check whether the implicit view is non-null. The real test is `EmbedderTest.ImplicitViewNotNull`.

> But this is only because we don't have an option to control it right?

Yup that's correct.

> Should we add a TODO or at least comment to explain that this test might (should) be broken once we allow non-implicitView?

Good idea, added a TODO to `EmbedderTest.ImplicitViewNotNull`.